src/box/lua/net_box.lua

```diff
@@ -469,6 +506,7 @@ end
 
 local function stream_begin(stream, txn_opts, netbox_opts)
     check_remote_arg(stream, 'begin')
+    check_param_table(netbox_opts, REQUEST_OPTION_TYPES)
```

Comment:
Nit: I'd add a helper function `check_request_opts()`. Up to you. The patch looks good to me.

```Lua
local function check_request_opts(opts)
    check_param_table(opts, REQUEST_OPTION_TYPES)
end
```

Reply:
I don't think it's a great idea. If it was one place, where all opts could be checked, then it would be reasonable. However, we check them with `check_param_table`, in high-level functions (e.g. `conn:ping()`) and after that even in the `_request'. All of this checks scattered throughout the code and the introduction of method, you proposed, could mislead the programmer, who will read all of this. Let's stay with the current version)